manifests/init.pp

```diff
@@ -72,6 +72,7 @@
   $qpid_router_broker_port   = $capsule::params::qpid_router_broker_port,
 ) inherits capsule::params {
 
+  include ::apache
```

Comment:
I'm leaning to getting rid of ```capsule::config``` and move it to init.pp. Might as well get rid of ```capsule::install``` while you're at it. They both appear to be leftovers from when this module did a lot more than it used to.

@ehelms do you agree?